src/index.js

```diff
@@ -0,0 +1,101 @@
+const skyLeft = {
+  Sunny: '☀️🌤🌞🌤☀️',
+  Cloudy: '🌫☁️🌥⛅️🌤',
+  Rainy: '🌈⛈🌧🌧🌦',
+  Snowy: '🌨❄️🌨❄️🌨',
+};
+
+const skyRight = {
+  Sunny: '☀️🌤🌞🌤☀️',
+  Cloudy: '🌤⛅️🌥☁️🌫',
+  Rainy: '🌦🌧🌧⛈💧',
+  Snowy: '🌨❄️🌨❄️🌨',
+};
+
+const skyColor = {
+  Sunny: 'skyblue',
+  Cloudy: 'lightgrey',
+  Rainy: 'lightblue',
+  Snowy: 'lightsteelblue',
+};
+
+const skySelector = document.getElementById('sky__selector');
+const leftSky = document.getElementById('sky__left');
+const rightSky = document.getElementById('sky__right');
+
+const defaultSky = skySelector.options[skySelector.selectedIndex].text;
+
+leftSky.textContent = skyLeft[defaultSky];
+rightSky.textContent = skyRight[defaultSky];
+
+const backgroundColor = document.getElementsByClassName('weather__garden')[0];
+
+backgroundColor.style.backgroundColor = skyColor['Sunny'];
+
+skySelector.addEventListener('change', function () {
+  const currentSky = skySelector.options[skySelector.selectedIndex].text;
+  leftSky.textContent = skyLeft[currentSky];
+  rightSky.textContent = skyRight[currentSky];
+  backgroundColor.style.backgroundColor = skyColor[currentSky];
+});
```

Comment:
Rather than providing this function as an anonymous function that can only be used when the selector changes, we could create a named function that is callable from other locations. Then, we could use this function rather than the bare code on lines 26 through 33.